graylog2-web-interface/src/pages/LoggedInPage.jsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import AppRouter from 'routing/AppRouter';
+import CurrentUserPreferencesProvider from '../contexts/CurrentUserPreferencesProvider';
 
-class LoggedInPage extends React.Component {
-  render() {
-    return <AppRouter />;
-  }
-}
+const LoggedInPage = () => (
+  <CurrentUserPreferencesProvider>
+    <AppRouter />
+  </CurrentUserPreferencesProvider>
+);
```

Comment:
One thought regarding the placement, which does not require a change right now: At some point it will be very useful to wrap the `<GraylogThemeProvider>` with the `<CurrentUserPreferencesProvider>` to show a specific theme based on the user settings. But we can still adjust this when we have the need.

Reply:
Good point!